cpp/trtorchexec/main.cpp

```diff
@@ -57,12 +57,13 @@ int main(int argc, const char* argv[]) {
 
   auto compile_spec = trtorch::CompileSpec(dims);
   compile_spec.workspace_size = 1 << 24;
+  compile_spec.sparse_weights = true;
 
-  std::cout << "Checking operator support" << std::endl;
-  if (!trtorch::CheckMethodOperatorSupport(mod, "forward")) {
-    std::cerr << "Method is not currently supported by TRTorch" << std::endl;
-    return -1;
-  }
+  // std::cout << "Checking operator support" << std::endl;
```

Comment:
Why is this commented out?

Reply:
This file shouldn't have any changes. Will fix now.

Reply:
done